src/test/java/org/opentripplanner/apis/gtfs/GraphQLScalarsTest.java

```diff
@@ -1,22 +1,51 @@
 package org.opentripplanner.apis.gtfs;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.params.provider.Arguments.of;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
+import graphql.language.StringValue;
 import graphql.schema.CoercingSerializeException;
 import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 import org.locationtech.jts.geom.Coordinate;
 import org.locationtech.jts.geom.GeometryFactory;
+import org.opentripplanner._support.time.ZoneIds;
 import org.opentripplanner.framework.json.ObjectMappers;
 
 class GraphQLScalarsTest {
```

Comment:
Consider making a test class per Scalar, at least put your new scalar in its own test class.

Reply:
I separated the tests in https://github.com/opentripplanner/OpenTripPlanner/pull/5660/commits/84083b6476ad1afd52d48405e024838751ecfef9